### PR TITLE
To err is human ...

### PR DESCRIPTION
**Pull request by arobson**
But to render them properly is autohost. Recent changes in hyped's rendering pipeline caused an incompatibility with how errors are rendered with autohost's http envelope implementation.

This PR addresses that. I changed the default error handling behavior from sending back the error message by default (too dangerous/risky/easy-to-forget-to-change) and now return `{ "message": "Server error" }` as a way to protect the app and encourage devs to change it to something meaningful if warranted. I also removed the demo junk and added a `method` property to the envelope.

### CODE REVIEW

File: spec/behavior/httpEnvelope.spec.js [hunk 1]
```diff
@@ -7,13 +7,14 @@ describe( 'HTTP Envelope', function() {
 
 	describe( 'when handling return', function() {
 
-		describe( 'with host defined custom error', function() {
-			var envelope, req, res, request, host;
+		describe.only( 'with host defined custom error', function() {
```
Comment (dcneiner) [inline spec/behavior/httpEnvelope.spec.js]: If only this could be removed...

**Comment by arobson** [comment]
@dcneiner - thanks for catching that!

**Comment by Prestaul** [comment]
@arobson, am I to understand that you got within 39 lines of 14k and still fell short?  So disappointing...

**Comment by arobson** [comment]
@Prestaul lol, I'm sorry, I shall try to delete more heavily next time ;)

**Pull request merged**